src/sentry/static/sentry/app/views/organizationDiscover/aggregations/index.tsx

```diff
@@ -1,33 +1,35 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import {Box} from 'grid-emotion';
-
-import Link from 'app/components/links/link';
-import InlineSvg from 'app/components/inlineSvg';
+import * as React from 'react';
+import styled from 'react-emotion';
 import {t} from 'app/locale';
+import space from 'app/styles/space';
 
-import Aggregation from './aggregation';
 import {PlaceholderText, SelectListItem, AddText, SidebarLabel} from '../styles';
+// eslint-disable-next-line no-unused-vars
```

Comment:
It's super annoying this is needed, not sure if it's a bug in the typescript-eslint parser

Reply:
Related to https://github.com/eslint/typescript-eslint-parser/issues/457

Reply:
Ref: https://github.com/typescript-eslint/typescript-eslint/issues/363

Reply:
and this https://github.com/typescript-eslint/typescript-eslint/issues/46#issuecomment-470486034

Reply:
recommendation here https://43081j.com/2019/02/using-eslint-with-typescript